config/fcos/v1_6_exp/validate.go

```diff
@@ -51,3 +51,19 @@ func (user GrubUser) Validate(c path.ContextPath) (r report.Report) {
 	}
 	return
 }
+
+func (c Config) Validate() (r report.Report) {
+	for _, disk := range c.Storage.Disks {
+		if disk.Device == "/dev/disk/by-id/coreos-boot-disk" {
+			continue
+		}
+		if disk.WipeTable != nil && !*disk.WipeTable {
```

Comment:
We have a package called "util" which does nil check for ya, it should look like =>
`util.IsFalse(disk.WipeTable)`